src/nrniv/CMakeLists.txt

```diff
@@ -250,13 +250,16 @@ endif()
 # All source directories to include
 # =============================================================================
 add_library(nrniv_lib SHARED ${NRN_NRNIV_LIB_SRC_FILES})
-if(READLINE_FOUND)
+if(NRN_WINDOWS_BUILD)
+  target_link_libraries(nrniv_lib ${TERMCAP_LIBRARIES} ${Readline_LIBRARY})
+elseif(READLINE_FOUND)
   target_link_libraries(nrniv_lib ${Readline_LIBRARY})
 else()
   target_link_libraries(nrniv_lib readline)
 endif()
 
-if(CURSES_FOUND)
+if(NRN_WINDOWS_BUILD)
+elseif(CURSES_FOUND)
```

Comment:
just a nitpick: rather than empty if statement, we wrap wrap CURSES_FOUND and TERMCAP_FOUND blocks under :


```
if(NOT NRN_WINDOWS_BUILD)

endif()
```

Reply:
agree. will make the change